game_patch/misc/vpackfile.cpp

```diff
@@ -529,6 +613,47 @@ static void vpackfile_cleanup_new()
     g_packfiles.clear();
 }
 
+static void load_vpp_files_from_directory(const char* files, const char* directory)
```

Comment:
first argument name is not very clear. Also someone calling this function must know to use `*.vpp` pattern and not like `*.*` or it will try to load non-vpp files as vpps, despite the function claiming to load only vpps from directory. Also directory name is repeated in both args. A clearer API would be to build to build the pattern in the function using `directory` argument